jest.conf/visual.testUtils.js

```diff
@@ -6,30 +6,15 @@ import percySnapshot from '@percy/puppeteer';
  * @param {string} component - The name of the Vue component to render.
  * @param {Object} props - The props to pass to the component.
  * @param {Object} [slots={}] - An object representing the slots to pass to the component.
- * The structure of the `slots` object should be as follows:
- *
- * Example:
- * {
- *   default: {
- *     element: "div", // or any Vue component like KIcon
```

Comment:
Lets revert this change, please :). JsDocs are super super important for methods like this.